samples/java-valueentity-counter-pojo/src/main/proto/com/example/counter_api.proto

```diff
@@ -0,0 +1,50 @@
+// This is the public API offered by your entity.
+syntax = "proto3";
+
+package com.example; // <1>
+
+import "akkaserverless/annotations.proto"; // <2>
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+option java_outer_classname = "CounterApi"; // <3>
+
+message IncreaseValue { // <4>
+  string counter_id = 1 [(akkaserverless.field).entity_key = true]; // <5>
+  int32 value = 2;
+}
+
+message DecreaseValue {
+  string counter_id = 1 [(akkaserverless.field).entity_key = true];
+  int32 value = 2;
+}
+
+message ResetValue {
+  string counter_id = 1 [(akkaserverless.field).entity_key = true];
+}
+
+message GetCounter {
+  string counter_id = 1 [(akkaserverless.field).entity_key = true];
+}
+
+message CurrentCounter { // <6>
+  int32 value = 1;
+}
+
+service CounterService { // <7>
+  option (akkaserverless.codegen) = { // <8>
+    value_entity: { // <9>
+      name: "com.example.domain.Counter" // <10>
+      entity_type: "counter" // <11>
+
+      // NOTE: domain classed are in org.example because 
+      // the CI tests remove the generated classes in src/java/main/com
```

Comment:
That would be somewhat confusing if we use this sample in docs or as user example. Can we tweak CI?

Reply:
Yep, we can change it. We can remove the Main and the Entity explicitly instead of deleting the parent dir.